src/libs/actions/Report.ts

```diff
@@ -767,10 +767,13 @@ function openReport(
         return;
     }
 
-    const optimisticReport = reportActionsExist(reportID)
-        ? {}
+    const optimisticReport: Report = reportActionsExist(reportID)
+        ? {
+              reportID,
+          }
```

Comment:
this change seems not safe to me. I think this change is unnecessary if we apply [this approach](https://github.com/Expensify/App/issues/51488#issuecomment-2443758461). Anyways, could we handle it in https://github.com/Expensify/App/issues/51488 and revert this change? Thanks